include/quicly.h

```diff
@@ -457,6 +479,7 @@ struct _st_quicly_conn_public_t {
          */
         quicly_cid_t offered_cid;
         struct st_quicly_conn_streamgroup_state_t bidi, uni;
+        uint32_t n_retired_cids;
```

Comment:
I believe it is in your ToDo list to create a structure that maintains the list of sequence numbers of active connection IDs, because without that, we cannot see if the RCID frame that we receive is a duplicate of a previously received frame.

Reply:
Yes, it's coming in the next round of PR. What I'm thinking right now is to have an array like `int retired_cids[MAX_PATH_ID];` to indicate which one has been retired.

Reply:
Makes sense. I'll go with `uint64_t` as I also hate the maintenance burden with `uint8_t`.